src/main/java/bobby/manager/FileManager.java

```diff
@@ -0,0 +1,87 @@
+package bobby.manager;
+import bobby.task.Deadline;
+import bobby.task.Event;
+import bobby.task.Task;
+import bobby.task.ToDo;
+
+import java.io.File;
+import java.util.ArrayList;
+import java.util.Scanner;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public abstract class FileManager {
+    public static final String FILEPATH = "data/bobby.txt";
+
+    public static void createFile(File f) throws IOException {
+        f.getParentFile().mkdirs();
+        f.createNewFile();
+    }
+
+    public static Task parseDataToTask(String[] splitData) {
+        String taskType = splitData[0].toLowerCase();
+        String taskDoneStatus = splitData[1];
+        String taskDescription = splitData[2].trim();
+        String fullTaskDescription;
+
+        Task task = new Task(taskDescription);
+
+        switch (taskType) {
+        case "todo":
+            task = new ToDo(taskDescription);
+            break;
+        case "deadline":
+            String by = splitData[3].trim();
+            fullTaskDescription = String.format("%s /by %s", taskDescription, by);
+            task = new Deadline(fullTaskDescription);
+            break;
+        case "event":
+            String at = splitData[3].trim();
+            fullTaskDescription = String.format("%s /at %s", taskDescription, at);
+            task = new Event(fullTaskDescription);
+            break;
+        default:
+            ResponseManager.printIOExceptionMessage();
+            break;
+        }
+
+        if (taskDoneStatus.equals("X")) {
+            task.markAsDone();
+        }
+
+        return task;
+    }
+
+    public static void bootUpData(String filePath, TaskManager taskManager) throws IOException {
+        File f = new File(filePath);
```

Comment:
Please consider using the "file" as the name of the File object instead of "f". Similar problems can be found from other variables.